docs/next/guides/upgrade-and-migration/migrating-from-9.0-to-10.0.md

```diff
@@ -0,0 +1,70 @@
+- [`beforeViewRender`](@/api/hooks.md#beforeviewrender)
+- [`afterViewRender`](@/api/hooks.md#afterviewrender)
+
+## Step 2: Adapt to the HyperFormula dependency update
+
+In Handsontable 10.0.0, we updated the optional HyperFormula dependency from `0.6.2` to `^1.1.0` ([#8669](https://github.com/handsontable/handsontable/pull/8669)).
+
+For more details on the breaking changes between HyperFormula 0.6.x and HyperFormula 1.0.x, see the [migration guide](https://handsontable.github.io/hyperformula/guide/migration-from-0.6-to-1.0.html).
+
+## Step 3: Adapt to the configuration options' new default values
+
+In Handsontable 10.0.0, we changed the default values of the [`autoWrapCol`](@/api/options.md#autowrapcol) and [`autoWrapRow`](@/api/options.md#autowraprow) [configuration options](@/guides/getting-started/setting-options.md), from `true` to `false` ([#8662](https://github.com/handsontable/handsontable/pull/8662)):
+
+| Before                                              | After                                                |
+| --------------------------------------------------- | ---------------------------------------------------- |
+| [`autoWrapCol: true`](@/api/options.md#autowrapcol) | [`autoWrapCol: false`](@/api/options.md#autowrapcol) |
+| [`autoWrapRow: true`](@/api/options.md#autowraprow) | [`autoWrapRow: false`](@/api/options.md#autowraprow) |
+
+We also changed the default values for the [`rowsLimit`](@/api/copypaste.md#rowslimit) and [`columnsLimit`](@/api/copypaste.md#columnslimit) options of the [`CopyPaste`](@/api/copypaste.md) plugin, from `1000` to `Infinity` ([#8676](https://github.com/handsontable/handsontable/pull/8676)):
+
+| Before                                                  | After                                                       |
+| ------------------------------------------------------- | ----------------------------------------------------------- |
+| [`rowsLimit: 1000`](@/api/copypaste.md#rowslimit)       | [`rowsLimit: Infinity`](@/api/copypaste.md#rowslimit)       |
+| [`columnsLimit: 1000`](@/api/copypaste.md#columnslimit) | [`columnsLimit: Infinity`](@/api/copypaste.md#columnslimit) |
+
+## Step 4: Adapt to the hook changes
+
+In Handsontable 10.0, we unified the naming of an argument used by the [`beforeOnCellMouseDown`](@/api/hooks.md#beforeoncellmousedown) and [`beforeOnCellMouseOver`](@/api/hooks.md#beforeoncellmouseover) hooks ([#8591](https://github.com/handsontable/handsontable/pull/8591)):
+
+| Before              | After        |
+| ------------------- | ------------ |
+| `blockCalculations` | `controller` |
+
+This change affects the following plugins:
+- [`ColumnSorting`](@/api/columnsorting.md)
+- [`ManualColumnMove`](@/api/manualcolumnmove.md)
+- [`ManualRowMove`](@/api/manualrowmove.md)
+- [`NestedHeaders`](@/api/nestedheaders.md)
+
+For more details, see [this PR](https://github.com/handsontable/handsontable/pull/8591).
+
+## Step 5: Adapt to the font changes
```

Comment:
I wonder if it would be useful to prove point 5 with a photo of new styles compared to the old ones, but it shouldn't block this merge.